### DISCUSSION TITLE
Wifiphisher not working with Alpha AWSU036ACH

### DISCUSSION THREAD

**Discussion by AndreR10** [wifiphisher/wifiphisher#1627] [Q&A]
Hello,

I'm trying to use Wifiphisher with my Alpha AWSU036ACH, but when I run the program it gets stuck on the "Timezone" section. 

![image](https://github.com/wifiphisher/wifiphisher/assets/45071609/8423b14e-f203-4152-ab41-6ebebaac3b80)

All the drivers are installed and the system is updated.

The network adapter works with other programs like aircrack-ng but I never get it to work with wifiphisher. I also tried running kali from a pen drive and notice that the wifiphisher works with my internal network card, but never with the Alpha. 

Does any one know how I can solve this or why this is happening?

**Comment by reddyvijay9555** [comment]
i am also facing same issue
